inlong-sort/sort-connectors/postgres-cdc/src/main/java/org/apache/inlong/sort/cdc/postgres/table/PostgreSQLReadableMetaData.java

```diff
@@ -426,8 +435,8 @@ private static StringData getCanalData(SourceRecord record, TableChange tableSch
     /**
      * convert debezium operation to canal-json operation type
      *
-     * @param record
-     * @return
+     * @param record: source record
```

Comment:
Hi, there are not need add `:` after `@param` and `@return`:

![image](https://user-images.githubusercontent.com/31994335/223605644-bb195f6d-8a95-44ea-b114-e80727048b89.png)

After javadoc compiles, it looks like:

![image](https://user-images.githubusercontent.com/31994335/223605737-41fd6691-9231-4887-a835-c71fe1b78331.png)